components/main_page/wikis/wildcard/main_page_layout_data.lua

```diff
@@ -0,0 +1,214 @@
+		heading = 'Matches',
+		body = '{{#invoke:Lua|invoke|module=Widget/Factory|fn=fromTemplate|widget=Match/Ticker/Container}}' ..
+			'<div style{{=}}"white-space:nowrap; display: block; margin:0 10px; ' ..
+			'font-size:15px; font-style:italic; text-align:center;">[[Liquipedia:Matches|See more matches]]</div>',
+		padding = true,
+		boxid = 1507,
+	},
+	tournaments = {
+		heading = 'Tournaments',
+		body = '{{#invoke:Lua|invoke|module=Widget/Factory|fn=fromTemplate|widget=Tournaments/Ticker' ..
+			'|upcomingDays=60|completedDays=20}}',
+		padding = true,
+		boxid = 1508,
+	},
+}
+
+return {
+	banner = {
+		lightmode = 'Wildcard full lightmode.svg',
+		darkmode = 'Wildcard full darkmode.svg',
+	},
+	metadesc = 'Comprehensive Wildcard wiki with articles covering everything from champions, to summons, ' ..
+	'to tournaments, to competitive players, and teams.',
+	title = 'The Wildcard Wiki',
+	navigation = {
+		{
+			file = 'Wildcard Characters 2.jpg ',
+			title = 'Champions',
+			link = 'Portal:Champions',
+			count = {
+				method = 'LPDB',
+				table = 'datapoint',
+				conditions = '[[type::character]]',
+			},
+		},
+		{
+			file = 'Wildcard Characters 1.jpg ',
+			title = 'Summons',
+			link = 'Portal:Summons',
+			count = {
+				method = 'LPDB',
+				table = 'datapoint',
+				conditions = '[[type::summons]]',
+			},
+		},
+		{
+			file = 'Wildcard Lushland Arena.jpg',
+			title = 'Maps',
+			link = 'Portal:Maps',
+			count = {
+				method = 'LPDB',
+				table = 'datapoint',
+				conditions = '[[type::map]]',
+			},
+		},
+		{
+			file = 'Wildcard Frostburn Arena.jpg',
+			title = 'Tournaments',
+			link = 'Portal:Tournaments',
+			count = {
+				method = 'LPDB',
+				table = 'tournament',
+			},
+		},
+		{
+			file = 'Wildcard Screenshot 1.jpg',
+			title = 'Teams',
+			link = 'Portal:Teams',
+			count = {
+				method = 'LPDB',
+				table = 'team',
+			},
+		},
+		{
+			file = 'Wildcard Screenshot 2.jpg',
+			title = 'Players',
+			link = 'Portal:Players',
+			count = {
+				method = 'LPDB',
+				table = 'player',
+			},
+		},
+	},
+	layouts = {
+		main = {
+			{ -- Left
+				size = 6,
+				children = {
+					{
+						mobileOrder = 1,
+						content = CONTENT.champions,
+					},
+					{
+						mobileOrder = 4,
+						content = CONTENT.transfers,
+					},
+					{
+						mobileOrder = 8,
+						content = CONTENT.wantToHelp,
+					},
+				}
+			},
+			{ -- Right
+				size = 6,
+				children = {
+					{
+						mobileOrder = 2,
+						children = {
+							{
+								children = {
+									{
+										noPanel = true,
+										content = CONTENT.filterButtons,
+									},
+								},
+							},
+							{
+								size = 6,
+								children = {
+									{
+										noPanel = true,
+										content = CONTENT.matches,
+									},
+								},
+							},
+							{
+								size = 6,
+								children = {
+									{
+										noPanel = true,
+										content = CONTENT.tournaments,
+									},
+								},
+							},
+						},
+					},
```

Comment:
If Esports is not a focus on the wiki at all for now (per your other PR), do we even want this?